### ISSUE TITLE
Agree field conditional Checked / Unchecked rule isn't available if setup from a Group field

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#500]
**Description**
When an Agree field is added within a Group field, and then a Single-line Text is added in this Group, the Condition to show this Single-line Text field based on the Agree field value can't be setup as Checked / Unchecked: a text value is expected instead.

If this Agree field is added outside of a Group field, the expected behaviour happens for the condition of another field based on the Checked / Unchecked value.

**Steps to reproduce**
1. Create a group field
2. Add an Agree field in this group
3. Add a Single-line Text field in this group
4. Save the form setup
5. Reload the setup page
6. Edit Single-line Text conditional
7. Try to add rule based on the Agree field Checked / Unchecked value

**Additional info**

Plugin version: 1.4.7
Craft version: 3.6.17
Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.8](https://github.com/verbb/formie/releases/tag/1.4.8)
